token-lending/program/src/state/reserve.rs

```diff
@@ -795,7 +799,8 @@ impl Pack for Reserve {
             8,
             8,
             PUBKEY_BYTES,
-            248
+            1,
+            247
```

Comment:
Don't forget to update the comments on 731
`const RESERVE_LEN: usize = 619; // 1 + 8 + 1 + 32 + 32 + 1 + 32 + 32 + 32 + 8 + 16 + 16 + 16 + 32 + 8 + 32 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 8 + 8 + 1 + 8 + 8 + 32 + 248`